### ISSUE TITLE
how to send activation email

### ISSUE THREAD

**Issue by pajooh** [arocks/edge#1]
currently, after sign-up user automatically get's logged in.
i think the most common use case is to send him an activation mail

**Comment by arocks** [comment]
True. But it would have the complexity of adding a workflow to handle the one-time activation links etc. Let me see if a simple implementation is possible.

**Comment by 42hrs** [comment]
Have you already considered integrating django-allauth?

**Comment by a113n** [comment]
I wanted to implement this as well based on the existing authtools framework but without success. According to the following reference link in the authtools github:

https://github.com/fusionbox/django-authtools/issues/34

@rockymeza mentioned that there's an example for the admin creation of users:
http://django-authtools.readthedocs.org/en/latest/how-to/invitation-email.html

Any idea on how to extend that for normal user registration as well?

**Comment by rockymeza** [comment]
Hi,

Sorry to budge in, you may be interested in [django-registration](https://django-registration.readthedocs.org/en/2.0.4/index.html). Even if you don't end up using it, it could give you some ideas.
